package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "camelcase-keys",
-	"version": "7.0.2",
+	"version": "7.1.0",
```

Comment:
Don't bump version in pull requests. It's up to the maintainer to decide when and how to do this (applies to all pull requests on GitHub).

Reply:
done.